application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -90,6 +96,10 @@ private void handleHelpThreadCreated(ThreadChannel threadChannel) {
             .queueAfter(5, TimeUnit.SECONDS);
     }
 
+    private static boolean isPostedByBot(Message message) {
+        return message.getAuthor().isBot();
```

Comment:
that also triggers for other bots. instead, use self-user. sth like `message.getAuthor().isSelfUser()` (lookup JDA docs for details)

Reply:
Would you like to provide a link to help me out, because i didn't find any `isSelfUser()` by googling.

Reply:
`jda.getSelfUser()`